src/account/ProfileSubmenu.test.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import 'react-native'
+import { Provider } from 'react-redux'
+import ProfileSubmenu from 'src/account/ProfileSubmenu'
+import { Screens } from 'src/navigator/Screens'
+import MockedNavigator from 'test/MockedNavigator'
+import { FetchMock } from 'jest-fetch-mock/types'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+import { mockE164Number } from 'test/values'
+import networkConfig from 'src/web3/networkConfig'
+import { act, fireEvent, render, waitFor } from '@testing-library/react-native'
+import * as Keychain from 'react-native-keychain'
+const mockFetch = fetch as FetchMock
+const mockedKeychain = jest.mocked(Keychain)
+import Logger from 'src/utils/Logger'
+
+jest.mock('src/utils/Logger')
+
+mockedKeychain.getGenericPassword.mockResolvedValue({
+  username: 'some username',
+  password: 'someSignedMessage',
+  service: 'some service',
+  storage: 'some string',
+})
+
+describe('ProfileSubmenu', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('shows the expected menu items', () => {
+    const store = createMockStore()
+    const { queryByTestId } = render(
+      <Provider store={store}>
+        <MockedNavigator component={ProfileSubmenu}></MockedNavigator>
+      </Provider>
+    )
+    expect(queryByTestId('ProfileSubmenu/EditProfile')).toBeTruthy()
```

Comment:
probably this is ported over but using the `getBy*` queries would be nice for truthy assertions